cdisc_rules_engine/sql_operations/dataset_column_order.py

```diff
@@ -0,0 +1,36 @@
+from cdisc_rules_engine.constants.metadata_columns import METADATA_COLUMNS
+from cdisc_rules_engine.models.sql_operation_result import SqlOperationResult
+from cdisc_rules_engine.sql_operations.sql_base_operation import SqlBaseOperation
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SqlDatasetColumnOrderOperation(SqlBaseOperation):
+
+    def _execute_operation(self):
+        dataset_name = self.params.domain.lower()
+        all_tables = self.data_service.pgi.schema.get_tables()
+
+        dataset_schema = next((schema for name, schema in all_tables if name == dataset_name), None)
+        if dataset_schema is None:
+            raise ValueError(f"Dataset '{dataset_name}' not found in schema.")
+
+        columns = dataset_schema.get_columns()
```

Comment:
Are we guaranteeing order in the schema? I definitely didn't do it intentionally if we are, which is a bit scary. We might want to save the column order specifically in a separate list